estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,28 @@
+
+from odoo import fields, models
+from odoo.api import depends, onchange
```

Comment:
We almost never import from `odoo.api` directly. You should avoid doing so to stay consistent with the standard codebase.